source/crud/tests/unified/estimatedDocumentCount.json

```diff
@@ -0,0 +1,302 @@
+{
+    "description": "estimatedDocumentCount",
+    "schemaVersion": "1.0",
+    "createEntities": [
+        {
+            "client": {
+                "id": "client0",
+                "observeEvents": [
+                    "commandStartedEvent"
+                ]
+            }
+        },
+        {
+            "database": {
+                "id": "database0",
+                "client": "client0",
+                "databaseName": "edc-tests"
+            }
+        },
+        {
+            "collection": {
+                "id": "collection0",
+                "database": "database0",
+                "collectionName": "coll0"
+            }
+        },
+        {
+            "collection": {
+                "id": "collection1",
+                "database": "database0",
+                "collectionName": "coll1"
+            }
+        }
+    ],
+    "initialData": [
+        {
+            "collectionName": "coll0",
+            "databaseName": "edc-tests",
+            "documents": [
+                {
+                    "_id": 1,
+                    "x": 11
+                },
+                {
+                    "_id": 2,
+                    "x": 22
+                },
+                {
+                    "_id": 3,
+                    "x": 33
+                }
+            ]
+        }
+    ],
+    "tests": [
+        {
+            "description": "estimatedDocumentCount uses $collStats on 4.9.0 or greater",
+            "runOnRequirements": [
+                {
+                    "minServerVersion": "4.9.0"
+                }
+            ],
+            "operations": [
+                {
+                    "name": "estimatedDocumentCount",
+                    "object": "collection0",
+                    "expectResult": 3
+                }
+            ],
+            "expectEvents": [
+                {
+                    "client": "client0",
+                    "events": [
+                        {
+                            "commandStartedEvent": {
+                                "command": {
+                                    "aggregate": "coll0",
+                                    "pipeline": [
+                                        {
+                                            "$collStats": {
+                                                "count": {
```

Comment:
The spec says this should be:

```
"count": {}
```

Reply:
You're right... 🤦  Changed to `"count": {}` in all occurrences.